coins/coins.go

```diff
@@ -220,19 +234,24 @@ func (a *WeiAmount) AsEtherString() string {
 	return a.AsEther().Text('f')
 }
 
-// AsStandard is an alias for AsEther, returning the Wei amount as ETH
-func (a *WeiAmount) AsStandard() *apd.Decimal {
+// AsStd is an alias for AsEther, returning the Wei amount as ETH
+func (a *WeiAmount) AsStd() *apd.Decimal {
```

Comment:
why the rename?

Reply:
This PR added `AsStandardString` to the `EthAssetAmount` interface's API, and that original name seemed too long.  Just using `String()` was causing issues, as `WeiAmount` implements the interface, but doesn't default to standard units.  Using `amt.AsStandard().Text('f')` was also too long and prone to developer mistakes or non-use.  STD is an acronym that most people are familiar with and wouldn't be confused with anything else, so I thought it was a good place to use it. `.AsStandard()` would be short enough without the acronym, but I wanted all the functions on the `EthAssetAmount` interface to be consistent in their naming. Having `AsStdString()` and `AsStandard()` on the same interface seemed inconsistent.